api/internal/conf/conf.go

```diff
@@ -191,22 +192,35 @@ func setConf() {
 		if config.Conf.Log.ErrorLog.FilePath != "" {
 			ErrorLogPath = config.Conf.Log.ErrorLog.FilePath
 		}
+
+		// access log
+		if config.Conf.Log.AccessLog.FilePath != "" {
+			AccessLogPath = config.Conf.Log.AccessLog.FilePath
+		}
+
 		if !filepath.IsAbs(ErrorLogPath) {
+			if strings.HasPrefix(ErrorLogPath, "winfile") {
+				return
+			}
 			ErrorLogPath, err = filepath.Abs(filepath.Join(WorkDir, ErrorLogPath))
 			if err != nil {
 				panic(err)
 			}
-		}
-
-		// access log
-		if config.Conf.Log.AccessLog.FilePath != "" {
-			AccessLogPath = config.Conf.Log.AccessLog.FilePath
+			if runtime.GOOS == "windows" {
+				ErrorLogPath = `winfile:///` + ErrorLogPath
+			}
 		}
 		if !filepath.IsAbs(AccessLogPath) {
+			if strings.HasPrefix(AccessLogPath, "winfile") {
```

Comment:
AFAIK the go package `path` handles the windows path normally. Why just use functions provided by it? Like `path.Abs`.

Reply:
- First of all, the function of getting the absolute path of a file is the original code, and I don't fully understand why I do it.
After my test, I found if I don't get the absolute path here, I can't test it pass on Linux. But using relative path can work normally on Windows.

- The purpose of detecting winfile prefix here is to define absolute path on Windows system in `conf.yaml`. `uber-go/zap` cannot handle paths that start with Windows drive letters such as `C:\`.